frontend/src/app/modules/work-space/dashboard/dashboard.component.ts

```diff
@@ -26,6 +26,7 @@ export class DashboardComponent extends BaseComponent
   }
 
   ngOnInit(): void {
+    this.currentUser = JSON.parse(localStorage.getItem(`user`));
```

Comment:
Use service instead. Don't need to check if a user exists since the page opens only for an authorized user.